wsgidav/dc/pam_dc.py

```diff
@@ -37,6 +38,8 @@ def __init__(self, wsgidav_app, config):
         self.pam_service = dc_conf.get("service", "login")
         self.pam_encoding = dc_conf.get("encoding", "utf-8")
         self.pam_resetcreds = dc_conf.get("resetcreds", True)
+        self.allow_users = dc_conf.get("allow_users", "all") # "all", "current" or list of allowed users
```

Comment:
We should validate the possible values/types

Reply:
Fixed!